src/app/applications/multi_call/page.mdx

```diff
@@ -0,0 +1,108 @@
+export const metadata = {
+  title: "Multi Call • Stylus by Example",
+  description:
+    "An example implementation of the Multi Call contract in Rust using Arbitrum Stylus.",
+};
+
+{/* Begin Content */}
+
+# Multicall
+
+An Arbitrum Stylus version implementation of [Solidity Multi Call contract](https://solidity-by-example.org/app/multi-call/) that aggregates multiple queries using a for loop and RawCall.
+
```

Comment:
In all other examples, we include the interface of the contract. We should include it here for consistency.